doc/user-guide.adoc

```diff
@@ -110,6 +110,7 @@ Each `RuntimeComponent` CR must at least specify the `applicationImage` paramete
 | `volumes` | A YAML object that represents a link:++https://kubernetes.io/docs/concepts/storage/volumes++[pod volume].
 | `volumeMounts` | A YAML object that represents a link:++https://kubernetes.io/docs/concepts/storage/volumes/++[pod volumeMount].
 | `storage.size` | A convenient field to set the size of the persisted storage. Can be overridden by the `storage.volumeClaimTemplate` property.
+| `storage.className`| A convenient field to set the storage class of the persisted storage. Can be overridden by the `storage.volumeClaimTemplate` property.
```

Comment:
Since these changes are not in a released driver, we can't update the doc/* yet. Please update this doc instead: https://github.com/application-stacks/runtime-component-operator/blob/main/internal/doc/draft-user-guide-v1.adoc